README.md

```diff
@@ -17,5 +17,9 @@
 | 7 | Юрий Копытов    | Просто кровь                           |
 | 8 | Алекса Бобина   | Вечное сияние чистого разума           |
 | 9 | Юля Филонова    | Мне не больно                          |
-|10 | Наташа Гвилава  | Запределье                             |
-|11 | Лера Пахнева    | Одинокий мужчина                       |
\ No newline at end of file
+| 10 | Алекса Бобина  | Вечное сияние чистого разума           |
+| 11 | Настя Корбей   | Большой куш                            |
+| 12 | Наташа Гвилава | Запределье                             |
+| 13 | Наташа Гвилава | Запределье                             |
+| 14 | Лера Пахнева   | Одинокий мужчина                       |
```

Comment:
Why do you add Lera and Natasha to the end? Please add yourself to the end and let them stay there they are.